app/form_objects/support/bulk_allocation_form.rb

```diff
@@ -0,0 +1,13 @@
+class Support::BulkAllocationForm
+  include ActiveModel::Model
+
+  attr_accessor :school_urns
+
+  validates :school_urns, presence: { message: 'Tell us the schools with confirmed coronvirus restrictions' }
+
+  def urn_list
+    return [] if school_urns.empty?
+
+    school_urns.split("\r\n").reject(&:blank?)
```

Comment:
Worth getting rid of leading and trailing spaces?

```suggestion
    school_urns.split("\r\n").map(&:strip).reject(&:blank?)
```